web-animations-1/Overview.bs

```diff
@@ -3304,7 +3304,8 @@ To produce <a>computed keyframe offsets</a>, we define a procedure to
 
 <a>Computed keyframes</a> are produced using the following procedure.
 Note that this procedure is only performed on a <a>keyframe effect</a> having
-an <a>effect target</a> for which computed property values can be calculated.
+an <a>effect target</a> for which computed property values can be calculated,
+as otherwise all propereties are <a>not animatable</a>.
```

Comment:
Although, I'm not exactly sure what the purpose of this change is. How is it related to pseudo elements? What is the difference between saying the procedure is not applied and the properties are "not animatable"?

It's also a bit weird to say they are "not animatable" since that's something that is normally defined for the property itself. We should instead say something like they follow the same logic as not animatable properties or something. Once I understand the purpose of the change I can help suggest wording.

Reply:
good point